tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1400,6 +1400,28 @@ def test_tiny_specaugment_librispeech(self):
         # fmt: on
         self.assertTrue(torch.allclose(logits[0][0, 0, :30].cpu(), EXPECTED_LOGITS, atol=1e-4))
 
+    @slow
+    def test_generate_with_intial_prompt(self):
```

Comment:
Small typo in the function name. :-) But excellent work on adding these unit tests!